mergify_engine/actions/merge/action.py

```diff
@@ -164,9 +209,23 @@ def _get_commit_message(pull, mode="default"):
         )
 
         if found and message_lines:
+            env = jinja2.sandbox.SandboxedEnvironment()
```

Comment:
That's a little less powerful because it'll raise on the first unknown attribute and won't list all of them… but it's good enough and much simpler so I took that approach.